Vulkan-Pipeline-Assistant/mainwindow.cpp

```diff
@@ -95,6 +96,29 @@ void MainWindow::HandleShaderFileDialog(QLineEdit* field) {
         tr("Open File"), ".", tr("Shader Files (*.spv)")));
 }
 
+//TODO allow multiple viewports
+void MainWindow::HandleViewChangeApply(QVector<QLineEdit*> v) {
+    PipelineConfig& config = this->m_vulkan->GetConfig();
+    VkViewport& viewport = config.viewports[0];
+    viewport.x = float(std::atof(v.at(0)->text().toStdString().c_str()));
+    viewport.y = float(std::atof(v.at(1)->text().toStdString().c_str()));
+    viewport.width = float(std::atof(v.at(2)->text().toStdString().c_str()));
+    viewport.height = float(std::atof(v.at(3)->text().toStdString().c_str()));
+    viewport.minDepth = float(std::atof(v.at(4)->text().toStdString().c_str()));
+    viewport.maxDepth = float(std::atof(v.at(5)->text().toStdString().c_str()));
```

Comment:
Could QString::toFloat be used rather than converting to std string and c string then atof?

Reply:
Yes! Changed all instances of atof to toFloat in mainwindow.cpp